views/templates/hook/displayBlockProduct.tpl

```diff
@@ -29,12 +29,20 @@
                     {/if}
                 {/if}&nbsp;
             </span>
-            {if empty($block['description'])}
-              <p class="block-title" style="color:{$textColor};">{$block['title']}</p>
-            {else}
-              <span class="block-title" style="color:{$textColor};">{$block['title']}</span>
-              <p style="color:{$textColor};">{$block['description'] nofilter}</p>
-            {/if}
+            <span class="block-title" style="color:{$textColor};">
+              {if empty($block['title'])}
+                &nbsp;
```

Comment:
I barely need to use something like that in order to fix alignment problems, don't you think that there is a solution using CSS or refactoring the style?

Reply:
Agree. Empty `block-title` keeps a line by CSS `min-height` now.

Reply:
In fact, in a few line above [line 30](https://github.com/PrestaShop/blockreassurance/pull/423/files#diff-e4a2b2b619a8cb3aed63a875b3a1c9041b424e111aa35e899a34002f6f7729e1R30), the same trick is used - shows a space when `block['icon'] == 'undefined'`